search/routes.js

```diff
@@ -68,12 +68,13 @@ router.get('/:query/:pageNumber/:itemsPerPage', async (req, res, next) => {
             query
         };
 
+        req.log.info({ query, pageNumber, itemsPerPage }, 'Creating search service');
         const searchService = createSearchService({
             caseReferenceNumber: req.session?.caseReferenceNumber,
-            logger: res.log
+            logger: req.log
         });
 
-        const response = await searchService.getSearchResults(query, pageNumber, itemsPerPage);
+        const response = await searchService.getSearchResults(query, pageNumber, itemsPerPage, req);
```

Comment:
Avoid passing the whole request, pass what we need.

Why is the cookie needed for the request inside `getSearchResults`?